docs/editor/accessibility.md

```diff
@@ -70,179 +70,181 @@ In `settings.json` file, nest the following code inside the outermost curly brac
+Audio cues indicate if the current line has certain markers that include: errors, warnings, breakpoints, folded text regions, or inline suggestions.
 
-They are played when the primary cursor changes its line or the first time a marker is added to the current line. Audio cues are enabled automatically when a screen reader is attached, but can also be controlled by the settings `audioCues.*`.
+They are played when the primary cursor changes its line or the first time a marker is added to the current line. Audio cues are automatically turned on when a screen reader is attached, but can also be controlled by the settings `audioCues.*`.
 
-The command **Help: List Audio Cues** lists all available audio cues, lets you hear each audio cue as you move through the list, and review which cues are currently enabled.
+The command **Help: List Audio Cues** lists all available audio cues, previews each audio cue as you move through the list, and reviews which cues are currently turned on.
 
 ## Hover accessibility
 
-Some hovers cannot be hovered normally which makes them hard to use with screen magnifiers. To work around this, while a hover is active hold the `kbstyle(Alt)` or `kbstyle(Option)` key to "lock" it in place such that it doesn't hide when hovered. Releasing the key will unlock the hover so it acts like normal.
+Some hovers cannot be hovered normally, which makes them hard to use with screen magnifiers. To work around this, hold the `kbstyle(Alt)` or `kbstyle(Option)` key while a hover is active to "lock" it in place so that it won't hide when hovered. Release the key to unlock the hover.
 
 ## Current known issues
 
-VS Code has some known accessibility issues depending on the platform. Here's a [full list](https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility) of VS Code accessibility issues.
+VS Code has some known accessibility issues, depending on the platform. For the full list, go to [VS Code accessibility issues](https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility).
 
 ### macOS
 
-There is screen reader support for the editor with VoiceOver.
+The editor includes screen reader support for VoiceOver.
 
 ### Linux
 
-VS Code works well with the Orca screen reader. If on your Linux distribution Orca does not read the editor content:
+VS Code works well with the Orca screen reader. If Orca in your Linux distribution does not read the editor content:
 
-* Make sure to have the setting `"editor.accessibilitySupport": "on"` in VS Code. You can do this using settings, or by running the **Show Accessibility Help** command and pressing `kbstyle(Ctrl+E)` to turn on accessibilitySupport.
+* Check the `"editor.accessibilitySupport": "on"` setting in VS Code. You can do this in the **Settings** menu or by running the **Show Accessibility Help** command and pressing `kbstyle(Ctrl+E)` to turn accessibility support on.
```

Comment:
There isn't a Settings menu

Reply:
Users can update their settings, either using the Settings editor UI or directly in the settings.json file. This far along in the topic I think we can just say "settings"